benchmarks/coo_vs_csr_edge_index/csr_functions.py

```diff
@@ -0,0 +1,88 @@
+import mlx.core as mx
+import numpy as np
+from typing import Optional
+
+
+def get_src_dst_features_CSR(
```

Comment:
AFAIK the CSR format is not the same thing as having a (N, 2) edge_index. Is there any reason why you named it CSR? I would say it's just the shape of the COO matrix that changes.

Example of CSR matrix:

```python
indptr = np.array([0, 2, 3, 6])
indices = np.array([0, 2, 2, 0, 1, 2])
data = np.array([1, 2, 3, 4, 5, 6])
csr_matrix((data, indices, indptr), shape=(3, 3)).toarray()
array([[1, 0, 2],
       [0, 0, 3],
       [4, 5, 6]])
```

Reply:
Yes, you're right 💯  - it's a leftover of some other initial tests and I carried it over without paying attention. I'll rename them as `column` and `row`.